### ISSUE TITLE
Will not Grunt. Just says ###

### ISSUE THREAD

**Issue by kupoback** [dhategan/brackets-grunt#34]
I am using Windows 8.1, latest version of Brackets, and latest version of this extension.

It detects my functions in the Gruntfile.js I have in the root of the folder, however, after loading tasks, when I click on the correct function to execute, I just get the output of "###" There is no indication of it executing, and when I check the combined file grunt's merging, there are no changes.

**Comment by kupoback** [comment]
I get the following

    [node-log 2:13:28 PM] { '0': { [Error: spawn ENOENT] code: 'ENOENT', errno: 'ENOENT', syscall: 'spawn' } }

**Comment by peterflynn** [comment]
@dhategan it looks like the latest version of the extension is missing the `./node/node_modules` folder, so it will always fail to find Grunt.  Previous versions of the extension included Grunt in that folder in the ZIP package.

**Comment by dhategan** [comment]
@peterflynn  Yes you are right. Though if you download it from the brackets registry web site ( or use the brackets interface to install it) you will have the node_modules folder. I'll try to make some time to fix some of the known issues but I don't want to promise anything yet.

**Comment by kupoback** [comment]
@dhategan @peterflynn  This still didn't resolve anything for me.

**Comment by kostyasorokin** [comment]
Exactly the same problem =((

> NodeDebugUtils.js:117 [node-log 8:07:29 PM] { '0': { [Error: spawn ENOENT] code: 'ENOENT', errno: 'ENOENT', syscall: 'spawn' } }
